ALSpringBackend/src/main/java/com/ardaslegends/data/domain/ProductionSiteType.java

```diff
@@ -0,0 +1,16 @@
+package com.ardaslegends.data.domain;
+
+public enum ProductionSiteType {
```

Comment:
Bank, harbour, house of healing and embassy are missing (they are under prod sites in the bible)

Reply:
Yep indeed, they are under ProductionSites in the bible but we don't really need to adhere to it since a production site in our domain is fundamentally different to what it is defined as in the bible.
- A production site in our domain is an entity that actually produces something a resource
- Special Buildings only have an effect

The only thing that's a bit suspicious is the "Workshop" type in production sites. The reason we put it in was, a workshop only provides the sieges that are built in it. So it provides a resource, namely different sieges.

Harbors, etc, have an effect which is why they are just mapped as an enum without fields.